queue/queue_test.go

```diff
@@ -783,6 +797,40 @@ waitLoop:
 	assert.Equal(t, size.Size, stats.Completed)
 }
 
+/* kim: TODO: test
```

Comment:
Should we resolve this?

Reply:
Oh yeah, I wrote the test, I just forgot to remove it.